cineast.json

```diff
@@ -1,9 +1,9 @@
 {
   "database": {
     "host": "localhost",
-    "selector": "COTTONTAIL",
-    "writer": "COTTONTAIL",
-    "port": 1865,
+    "selector": "POLYPHENY",
+    "writer": "POLYPHENY",
+    "port": 80,
```

Comment:
I don't think we want to make this the default, so this should be changed back to cottontail before being merged.